src/views/contacts/AddContact.js

```diff
@@ -263,6 +266,15 @@ export default class AddContact extends Component {
                     onChangeText={(text) => this.setState({ name: text })}
                   />
                 </Item>
+
+                <Item stackedLabel>
+                  <Label>{screenProps.t('Contacts:nodeAddress')}</Label>
+                  <Input
+                    value={this.state.nodeAddress}
+                    placeholder="/ip4/192.168.0.100/tcp/38191"
```

Comment:
perhaps `/ip4/0.0.0.0/tcp/0` to denote an unspecified address?

Reply:
this is an example, to guide the user on how to add the format